### ISSUE TITLE
Запоминать переход на полную версию сайти

### ISSUE THREAD

**Issue by GamePad64** [limonte/dear-habr#52]

**Comment by GamePad64** [comment]
Такое впечатление, что пофиксили. Уже второй день не вижу мобильной версии 👍

**Comment by GamePad64** [comment]
Нет, не пофикшено.

**Comment by bitre** [comment]
Должно работать – сейчас вешается кука на год.
Если можете предоставить какие-либо подробности, напишите, пожалуйста, в поддержку.

**Comment by GamePad64** [comment]
Это недавнее изменение? Потому что вчера мне ещё вылезала мобильная версия.

**Comment by bitre** [comment]
Нет, около полугода назад была жалоба в саппорт, на основании которой тогда и увеличили срок жизни куки.

**Comment by limonte** [comment]
@bitre этот тикет был создан 17 дней назад, после того как вы полгода назад сделали это изменение. Что-то очевидно сломалось и ваша кука не работает. Затестил, подтверждаю:

![](https://i.imgur.com/SKVxWiF.png)

**Comment by noscript** [comment]
> Должно работать – сейчас вешается кука на год.

Как бы не так, на сессию только. Перезапустил браузер и на тебе мобильная версия. Жутко бесит по несколько раз в день.

> Нет, около полугода назад была жалоба в саппорт, на основании которой тогда и увеличили срок жизни куки.

И потом какой-то мазохист попросил откатить?

> Если можете предоставить какие-либо подробности, напишите, пожалуйста, в поддержку.

В спорт лото еще можно написать... Куда еще подробней?
> ![](https://camo.githubusercontent.com/55f1cc9a342b38469acc332658fc572c28b08bb6/68747470733a2f2f692e696d6775722e636f6d2f534b56785769462e706e67)
